wordpress-mobile/WordPress-Android
# Update screenshot UI Tests with new designs

Updates the UI Tests that will be used for the screenshots project.

![android-phone-1](https://user-images.githubusercontent.com/1119271/93654970-e5665900-f9dd-11ea-8a6d-37bafb4f6b35.png)

#### Challenges

* Android screenshots are currently broken, there's a PR in progress at https://github.com/wordpress-mobile/WordPress-Android/pull/13031, and too get around it temporarily, I created a new branch and then did a `git pull origin fix/screenshot-ui-tests` to pull in the work from #13031.
* After I switch to the `fix/screenshot-ui-tests` branch, I still get one failed test with a wiremock error which I am not sure how to get around (see https://github.com/wordpress-mobile/WordPress-Android/pull/13031#pullrequestreview-501191009 for details), but I think updating mocked data will resolve it.
* I think I have a good draft for what's needed for the tests themselves with a few TODOs left but I am having trouble updating the mocked data and getting that to work so I can get to the screens I need to.

Updated mock data is a blocker at this point and I need help with that part.

To test:

1. Switch to this branch.
1. You may also need to pull in the `fix/screenshot-ui-tests` branch.
1. Select `WPScreensotTest` as the run configuration.
1. Run the test and confirm it passes.
PR submission checklist:
